packages/validator/README.md

```diff
@@ -0,0 +1,98 @@
+# `@airnode/validator`
+
+Package for json specifications validation. Validation process can be configured with validator templates.
+
+## Usage
+
+The validator can be run as an npm script, by providing template of specification, and the path to the JSON file of specification that will be validated:
+```sh
+npm run validate --template="[template]" --specs="[specsFile]"
+```
+
+Templates are case-insensitive, valid templates are: `config`, `OIS`, `apiSpecifications`/`apiSpecs` and `endpoints`. If arguments are in order `template`, `specs` command can be simplified: `npm run validate [template] [specsFile]`:
+```sh
+npm run validate config exampleSpecs/config.specs.json
+```
+
+Validator will automatically validate the latest available version of provided template, in case specific version should be used in validation, it can be passed as `--templateVersion` argument or as the last argument if they are ordered:
+```sh
+npm run validate config exampleSpecs/config.specs.json 0.1.0
+```
+
+Custom templates can be used, by providing path to the validator template file in place of `template`:
+```sh
+npm run validate templates/1.0.0/config.json exampleSpecs/config.specs.json
```

Comment:
Generally the default behavior is that relative path starts in the directory where the command is ran from, so I dont think it needs to be explicitly stated, otherwise it would have to be mentioned everywhere including the `specs` argument which is quite redundant tbh.